docs/installation/introduction.rst

```diff
@@ -84,128 +82,133 @@ offering the finest control over the entire platform:
 
-It is also possible to evolve an architecture after initial deployment, in case
-the underlying infrastructure also evolves (new machines can be added through
-the :doc:`expansion<./expansion>` mechanism, roles can be added or removed...).
+You can also evolve an architecture after initial deployment, if the underlying
+infrastructure also evolves (new machines can be added through the
+:doc:`expansion<./expansion>` mechanism, roles can be added or removed, etc.).
 
 
 Concepts
 ^^^^^^^^
-Although being familiar with
+
+Although familiarity with
 `Kubernetes concepts <https://kubernetes.io/docs/concepts/>`_
 is recommended, the necessary concepts to grasp before installing a MetalK8s
 cluster are presented here.
 
 Nodes
 """""
-:term:`Nodes <Node>` are Kubernetes worker machines, which allow running
-containers and can be managed by the cluster (control plane services,
-described below).
 
-Control Plane and Workload Plane
-""""""""""""""""""""""""""""""""
-This dichotomy is central to MetalK8s, and often referred to in other
-Kubernetes concepts.
+:term:`Nodes <Node>` are Kubernetes worker machines that allow running
+containers and can be managed by the cluster (see :ref:`control plane
+services<control_and_workload>`, next section).
+
+.. _control_and_workload:
+
+Control and Workload Planes
+"""""""""""""""""""""""""""
 
-The **control plane** is the set of machines (called :term:`nodes <Node>`) and
+The distinction between the control and workload planes is central to MetalK8s,
+and often referred to in other Kubernetes concepts.
+
+The **control plane** is the set of machines (called "nodes") and
 the services running there that make up the essential Kubernetes functionality
 for running containerized applications, managing declarative objects, and
-providing authentication/authorization to end-users as well as services.
-The main components making up a Kubernetes control plane are:
+providing authentication/authorization to end users as well as services. The
+main components of a Kubernetes control plane are:
 
 - :term:`API Server`
 - :term:`Scheduler`
 - :term:`Controller Manager`
 
-The **workload plane** indicates the set of nodes where applications
-will be deployed via Kubernetes objects, managed by services provided by the
-**control plane**.
+The **workload plane** is the set of nodes in which applications
+are deployed via Kubernetes objects, managed by services in the
+control plane.
 
 .. note::
 
    Nodes may belong to both planes, so that one can run applications
    alongside the control plane services.
 
 Control plane nodes often are responsible for providing storage for
-:term:`API Server`, by running :term:`etcd`. This responsibility may be
-offloaded to other nodes from the workload plane (without the ``etcd`` taint).
+API Server, by running :term:`etcd`. This responsibility may be
+offloaded to other nodes from the workload plane (without the etcd taint).
```

Comment:
Can we keep the monospace formatting on this value? It expresses a value to use, not just a meaning.

Reply:
Oh, actually you removed all of this formatting. Can we please revert this? I want to make sure it's obvious that those are not just "terms", but describe the actual value (omitting the common prefix) to input.

Reply:
In this instance, the topic of discussion is the taint, and "etcd" is the taint's name. Again, I may be rebelling against one of my predecessors' edicts here, but it's best to reserve monospace for CLI interactions only. If we are not asking the reader to enter a string into the terminal, or describing what the terminal returns, it should not be set monospace.